SocialCareCaseViewerApi/V1/Controllers/SocialCareCaseViewerApiController.cs

```diff
@@ -42,6 +42,14 @@ public class SocialCareCaseViewerApiController : Controller
         [Route("allocations")]
         public IActionResult GetAllocations([FromQuery] ListAllocationsRequest request)
         {
+            var validator = new ListAllocationsRequestValidator();
+            var validationResults = validator.Validate(request);
+
+            if (!validationResults.IsValid)
+            {
+                return BadRequest(validationResults.ToString());
```

Comment:
code comment above does not match http status response type

Reply:
Yeup, looking at it the comment was wrong anyways, looks like we return an empty list if there were no allocations found, replacing with just 200 and 400 response codes.